lib/config/error-codes/index.js

```diff
@@ -0,0 +1,69 @@
+/*
+ * Kuzzle, a backend software, self-hostable and ready to use
+ * to power modern apps
+ *
+ * Copyright 2015-2018 Kuzzle
+ * mailto: support AT kuzzle.io
+ * website: http://kuzzle.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const errors = require('kuzzle-common-objects').errors;
+const api = require('./api');
+
+/** Check the format of the error-codes config files
+ *  @param {object} - error config file domain
+ */
+
+function checkErrorCodes(errorCodes) {
+
+  for (const domain of Object.keys(errorCodes)) {
+    const domainName = errorCodes[domain];
+    
+    if (!domainName.hasOwnProperty('subdomains')) { throw new Error(`Error configuration file : Missing required 'subdomains' field in '${domain}'.`);} 
+    if (typeof domainName.subdomains !== 'object') { throw new Error(`Error configuration file : Field 'subdomains' must be an object in '${domain}'.`);} 
+    if (!domainName.hasOwnProperty('code')) { throw new Error(`Error configuration file : Missing required 'code' field in '${domain}'.`);} 
+    if (!Number.isInteger(domainName.code)) { throw new Error(`Error configuration file : Field 'code' must be an integer in '${domain}'.`);} 
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reformat the code to have only one statement per line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AOneStatementPerLine)